specification/security/resource-manager/Microsoft.Security/stable/2024-01-31/integrations.json

```diff
@@ -0,0 +1,737 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "Microsoft Defender for Cloud",
+    "description": "API spec for Microsoft.Security (Microsoft Defender for Cloud) resource provider",
+    "version": "2024-01-31"
```

Comment:
an API shouldn't be added directly as a stable GA API - there's no preview version of this resource type

Reply:
changed to preview